dm/config/subtask.go

```diff
@@ -274,6 +278,14 @@ func (c *SubTaskConfig) Adjust(verifyDecryptPassword bool) error {
 		return terror.ErrConfigOnlineSchemeNotSupport.Generate(c.OnlineDDLScheme)
 	}
 
+	if len(c.ShadowTableRule) == 0 {
+		c.ShadowTableRule = []string{"_gho", "_new"}
```

Comment:
```suggestion
		c.ShadowTableRule = []string{"^_(.+)_(?:new|gho)$"}
```
The pattern should contains exact one match group so we can use something like  `pattern.FindStringSubmatch` to extra the original table name from the pattern matching

Reply:
^_(.+)_(?:new|gho)$ , The user configuration feels not very friendly, Similar configuration,Meet three requirements at the same time，unless user set ["new","gho(.*)"], Background processing into ^_(.+)_(?:new|gho)$ 

there are three situations here:
1. Default, such as: _old
2. User-specified format, such as: _old_9999
3. User fuzzy matching format, such as: _old_9999(.*)

Reply:
i think  ["_new","_gho","_old(.*)"] is friendly